src/mscorlib/src/System/IO/Stream.cs

```diff
@@ -871,6 +821,23 @@ internal static void BlockingEndWrite(IAsyncResult asyncResult)
         {
             SynchronousAsyncResult.EndWrite(asyncResult);
         }
+        
+        internal void ValidateCopyToArguments(Stream destination, int bufferSize)
```

Comment:
Personally, I think it would be best to leave this `internal` since there are other subclasses of `Stream` that could benefit from this in a future PR, e.g. `MemoryStream` [here](https://github.com/dotnet/coreclr/blob/ebb839f3218a9410f76f4595574155304c5e101e/src/mscorlib/src/System/IO/MemoryStream.cs#L406).